src/Microsoft.AspNetCore.Server.Kestrel/KestrelServerOptions.cs

```diff
@@ -8,16 +8,33 @@ namespace Microsoft.AspNetCore.Server.Kestrel
 {
     public class KestrelServerOptions
     {
+        /// <summary>
+        /// Gets or sets whether the <c>Server</c> header should be included in each response.
+        /// </summary>
+        public bool AddServerHeader { get; set; } = true;
+
         public IServiceProvider ApplicationServices { get; set; }
 
         public IConnectionFilter ConnectionFilter { get; set; }
 
-        public bool NoDelay { get; set; } = true;
+        private int? _maxInputBufferLength = 1024 * 1024;
+        public int? MaxInputBufferLength
```

Comment:
Add doc comment. I think we might try to add this for all the KestrelServerOptions properties soon. https://github.com/aspnet/KestrelHttpServer/issues/922